shenyu-admin/src/main/resources/mappers/data-permission-sqlmap.xml

```diff
@@ -28,24 +28,34 @@
     </resultMap>
 
     <sql id="Base_Column_List">
-        id, date_created, date_updated, user_id, data_id, data_type
+        id, 
+        date_created, 
+        date_updated, 
+        user_id, 
+        data_id, 
+        data_type
     </sql>
 
     <delete id="deleteByUniqueKey">
-        delete from data_permission where data_id = #{dataId, jdbcType=VARCHAR}
-        and user_id = #{userId,jdbcType=VARCHAR} and data_type = #{dataType,jdbcType=INTEGER}
+        delete from data_permission 
```

Comment:
- The keywords  should be Uppercase;
-  Keep keywords right aligned；
-  Keep fields left aligned；